Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -583,9 +585,9 @@ internal string ExtractMatches(string source, string outputSeparator = null)
                 }
             }
 
-            private static string ExtractTextFromMatch(Match match)
+            private string ExtractTextFromMatch(Match match)

```

Comment:
>ExtractTextFromMatch [](start = 27, length = 20)

I'd rather make this `OutputFilter` component more general.  I think we should simply return `match.Value` if `GroupName` is `null` or `string.Empty`.

The caller to `OutputFilter`'s constructor is responsible for passing `ErrorMessageGroupName`.

Reply:
GroupName is currently never null or string.Empty.  It's optionally passed via one of the constructors to override the default ErrorMessageGroupName value.  That allows all the existing callers to remain unchanged.